src/main/webui/src/app/components/content/common/StoreListingWidget.jsx

```diff
@@ -32,8 +32,7 @@ LocalURLSection.propTypes = {
 const CapabilitiesSection = ({store}) => {
   if(store.type==="remote" || store.type==="hosted"){
     const options = Utils.storeOptions(store);
-    const styleClass = store.type==="remote"?"left-half":"right-half";
-    return <div className={styleClass}>
+    return <div className="left-half">
```

Comment:
Please don't change this because the alignment of this section is different between remote and hosted listing.

Reply:
removed